types.d.ts

```diff
@@ -67,3 +67,38 @@ type SettingsType = {
    smtp_username: string,
    smtp_password: string
 }
+
+type SearchResult = {
+   title: string,
+   url: string,
+   position: number,
+}
+
+export enum ScraperTypes {
```

Comment:
Can you please reverse scraper type definition changes you made? I am planning on refactoring the scraper so that anyone can add integration without messing up the scraper functions and only by defining an object. When you hardcode the types, it will only add an extra step for contributors to modify the scraperTypes enum.

Reply:
Sure :+1: I'll probably revert 16e5ae3c4acfc31149d722a4c19d3bdc2e00d27e and b5d6681 as these commits seem do more harm than good.

Reply:
Great! Waiting for the updated PR.